README.md

```diff
@@ -3,6 +3,12 @@
 [![Lint](https://github.com/HackSquadDev/contributor-of-the-xxx-python/actions/workflows/linting.yml/badge.svg)](https://github.com/HackSquadDev/contributor-of-the-xxx-python/actions/workflows/linting.yml)
 [![Format](https://github.com/HackSquadDev/contributor-of-the-xxx-python/actions/workflows/formatting.yml/badge.svg)](https://github.com/HackSquadDev/contributor-of-the-xxx-python/actions/workflows/formatting.yml)
 
+## Why?
+
+As a part of [HackSquad 2022](https://hacksquad.dev/), a competition took place where you'd have to create an autonomous "being of the Internet" (or "bot" to be precise) which takes the top contributor of a GitHub Organization within a given period and congratulates them for their contributions in both Discord and Twitter. \
+
+Speaking of that, This project has been made by one of the teams which joined forces in that competition. Feel free to have a look around! <br>
```

Comment:
Change:

```md
This project
```

To:

```md
this project
```

This is not the *start* of a line to use a capital letter.

Reply:
My bad that was a typo.

Reply:
@Omar8345 Done!